packages/react-shortcuts/src/Shortcut/Shortcut.tsx

```diff
@@ -1,9 +1,10 @@
 import * as React from 'react';
 import {contextTypes} from '../ShortcutProvider';
-import {Key} from '../types';
+import Key, {ModifierKey} from '../keys';
 
 export interface Props {
   keys: Key[];
+  modifierKeys?: ModifierKey[];
```

Comment:
Are we sure we never want to allow ordered key combos to be modifier keys or non-modifier-keys to be held?

I we don't have any shortcuts like that not then it's probably a good assumption. If we do we should consider changing the api to let you use all the keys for either and look something more like:

```TSX
type AnyKey = Key | ModifierKey;

interface ShortcutProps {
  ordered: AnyKey[];
  held: AnyKey[];
}

...

<Shortcut ordered={['f', 'Alt']} held={['Shift']} />
```

Reply:
I like the `ordered` and the `held` props, but I don't think we should allow any key to be held, because:

- it doesn't really make sense,
- could be a hacky solution, as `event.getModifierState` is only meant for checking if a modifier key is held down.

Reply:
👍